index.js

```diff
@@ -81,6 +81,20 @@ function ternaryDispatcher(wrap, f) {
   };
 }
 
+function fantasyLandUnaryDispatcher(wrap, f) {
+  return function unaryFluentureMethod(x) {
+    check (f, this);
```

Comment:
There is no strong need for type checking a Fantasy Land method, since it is assumed that the implementer follows the Fantasy Land contract. Or in other words; this method is not user-facing.